src/assets/i18n/en.json

```diff
@@ -1,8 +1,8 @@
 {
-  "CANCEL": "Cancel",
+  "CANCEL": "I'll save them first",
```

Comment:
Translation keys can be used in multiple locations in the application. This is an example. So when you change this to "I'll save them first", you are changing that in all places. I doubt that is what you want. You probably want to change it just for the editor-change confirmation dialog. So what you want is to create a new translation key and value in both language files and reference the new key inside the dialog.